catalog/views.py

```diff
@@ -97,7 +97,7 @@ def pgs(request, pgs_id):
 
     pub = score.publication
     citation = format_html(' '.join([pub.firstauthor, '<i>et al. %s</i>'%pub.journal, '(%s)' % pub.date_publication.strftime('%Y')]))
-    citation = format_html('<a target="_blank" href=https://doi.org/{}>{}</a>', pub.doi, citation)
+    citation = format_html('<a class ="external-link" target="_blank" href=https://doi.org/{}>{}</a>', pub.doi, citation)
```

Comment:
You shouldn't need to add the class `external-link`: it's added by the javascript.
However the HREF is missing quotes

Reply:
Oops - I did that because it wasn't working - good to know it's just the ""